lib/query_helper.rb

```diff
@@ -165,8 +165,8 @@ def view_query
   end
 
   def execute_query
-    query = build_query()
-    @results = @model.find_by_sql([query, @bind_variables]) # Execute Sql Query
+    query = to_sql
+    @results = @model.connection.select_all(query).as_json # Execute Sql Query
```

Comment:
@Girish2402  Do you have any test result to prove that it is not changing output, In my understanding `select_all` returns array of hash or json and `find_by_sql` returns model objects.

Reply:
Hi @vishalzambre @NileshN 

I add test cases for association(include) but its fails.
Association is failing for this code.
Do not merge changes.

Test results:
Failure/Error: ActiveRecord::Associations::Preloader.new.preload(payload, associations)

     NoMethodError:
       undefined method `_reflect_on_association' for Hash:Class